notebooks/WET/WET-007/WET-007_20241028_tie-vs-danish.ipynb

```diff
@@ -0,0 +1,344 @@
+{
```

Comment:
<p>Given that the <code>ts_aos_analysis</code> notebook does not have outputs, it may be a good idea ( beyond attaching PDF of the executed notebook in the PR,  and in the ticket)  to describe the result here in words (eg.  for Z7 Danish result of .. and TIE of ... with estimated uncertainty... agree with manual results ....) . But it is hard to describe results without any pictures, and with fleeting persistency of repos / collections / packages,    I wonder if we shouldn't have a repo where these notebooks are actually executed.. (<code>ts_aos_analysis_exec</code> ) ?</p><p> </p><p>Perhaps rewrite to avoid tautology, eg. <code>Note we employ means of combined Zernike estimates from each of the 9 detectors</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Bryce pointed out that we can't post executed notebooks anywhere public because many of the them contain embargoed data (hence why he deleted the PDFs in the PRs). We have created #aos-notebook-review on Slack to make the review process easier, but I agree we might want somewhere to store the executed notebooks. A private Github repo could work, but the size of the history will quickly get cumbersome. Maybe instead we create a shared Google drive folder where we can mirror the directory structure of the repo and drop PDFs of executed notebooks? It's a little cumbersome, but we could make adding the PDF to the drive be the last stage of the PR process?

Reply:
Yes, the slack channel is not a bad idea, but it may end up requiring too much scrolling with  "v1",  "better v1",  "better v1 after comments" .... If we can create a shared private google drive (or some other approved location) that would work better (in reply to review one can just overwrite).

```

Reply:
I think the place to have that is probably JIRA in the different tickets, also for traceability in the future for the project, it would be better than an ad-hoc solution that only AOS people can access

Reply:
@gmegh we cannot put compiled PDFs in tickets because they frequently contain on-sky pixels